build-logic/publication/src/main/kotlin/convention.sonatype.gradle.kts

```diff
@@ -32,9 +32,9 @@ val createStagingRepositoryTask: TaskProvider<CreateStagingRepositoryTask> = wit
         register<CreateStagingRepositoryTask>(createStagingTaskName) {
             group = "publication"
 
-            stagingProfileId.set(ossrhStagingProfileId)
-            user.set(ossrhUsername)
-            password.set(ossrhPassword)
+            stagingProfileId.set(ossrhStagingProfileId.get())
```

Comment:
Force to fail when read those pros because It was not clear why and when we loss them

Reply:
what if i don't have this properties locally? did you check that project syncing in IDE without it?

Reply:
I get, turn this file back
I see that only one option to force validation and be safe it's an additional flag. I don't want to create that flag now